src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -24,6 +24,7 @@
 class AppConfigMgmtScenarioTest(ScenarioTest):
 
     @ResourceGroupPreparer(parameter_name_for_location='location')
+    @AllowLargeResponse
```

Comment:
> In the latest run, the only test not generating file is this one - [test_appconfig_commands.py#L26-L160](https://github.com/pratiksanglikar/azure-cli/blob/prsangli/soft-delete/src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py#L26-L160) and it was run successfully - [test output](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1411036&view=logs&j=15e5eb7c-cf66-5b91-00a2-db9b11b5e0f9&t=845fa2ee-a602-59f5-4e4c-07ee6d0283b9&l=389) Seems like the issue with other tests has been resolved on it's own.

Hi @pratiksanglikar, this test case didn't actually run since `@AllowLargeResponse()` decorator is not properly used.
```suggestion
    @AllowLargeResponse()
```